src/SDKs/Azure.Core/data-plane/Azure.Core/Request.cs

```diff
@@ -28,6 +28,18 @@ public virtual void AddHeader(string name, string value)
 
         public abstract bool TryGetHeader(string name, out string value);
 
+        public abstract bool TryGetHeaderValues(string name, out IEnumerable<string> values);
```

Comment:
Why is it a try method vs simply returning an empty collection?

Reply:
The same reason we don't return null from TryGetHeader.